pkg/apis/istio/v1beta1/istio_types.go

```diff
@@ -451,7 +452,7 @@ type ProxyConfiguration struct {
 	// Options:
 	//   "" - disables access log
 	//   "/dev/stdout" - enables access log
-	// +kubebuilder:validation:Enum=,/dev/stdout
+	// +kubebuilder:validation:Enum=/dev/stdout
```

Comment:
```suggestion
	// +kubebuilder:validation:Enum=;/dev/stdout
```

I did not try, but I think the semicolon should be used here as the two valid values of this field are: "/dev/stdout" and "" (empty string).